src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -2734,11 +2771,15 @@ Status ColorControlServer::moveColorTempCommand(EndpointId endpoint,
  * @param commandData Struct containing the parameters of the command.
  * @return Status::Success when successful,
  *         Status::UnsupportedEndpoint when the provided endpoint doesn't correspond with a color XY transition state (verified in
- * moveToColorTemp function).
+ * moveToColorTemp function),
+ *         Status::ConstraintError when a command parameters is outside its defined value range.
  */
 Status ColorControlServer::moveToColorTempCommand(EndpointId endpoint,
                                                   const Commands::MoveToColorTemperature::DecodableType & commandData)
 {
+    // Command Parameters constraint checks:
+    VerifyOrReturnValue(commandData.transitionTime <= kMaxtransitionTime, Status::ConstraintError);
```

Comment:
`colorTemperatureMireds` needs a range check too.

Reply:
Ok, i can do that. The current implementation checks ColorTempPhysicalMinMireds and ColorTempPhysicalMaxMireds which are then used as limits for the move to final value.